src/services/queue/sendSqsMessage.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  SQSClient,
+  SendMessageRequest,
+  SendMessageCommand
+} from '@aws-sdk/client-sqs'
+import { getEnv } from '../../utils/helpers'
+
+export const sendSqsMessage = async (messageBody: object, queueUrl: string) => {
```

Comment:
Is it possible to be more specific about the messageBody type or is this something that might be expanded later?

Reply:
Yep, the idea behind this code is that it's completely generic, we could put anything in here (and almost certainly will). We could have written a really specific bit of code to send the message type we wanted to send for our case, but I knew there were more coming, so to avoid future people some boilerplate, this is what we came up with.